src/shared/components/home/site-form.tsx

```diff
@@ -1141,6 +1195,36 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
     this.setState(s => ((s.siteForm.legal_information = Some(val)), s));
   }
 
+  handleTaglineChange(i: SiteForm, index: number, val: string) {
+    let taglines = i.state.siteForm.taglines.unwrap();
+    taglines[index] = val;
+    i.state.siteForm.taglines = Some(taglines);
+    i.setState(i.state);
+  }
+
+  handleDeleteTaglineClick(
+    i: SiteForm,
+    index: number,
+    event: InfernoMouseEvent<HTMLButtonElement>
```

Comment:
I def agree with wanting to strictly type these events, but is MouseEvent correct? Maybe there's a ClickEvent. I'm thinking of phones and tablets that don't use mouse events.

Reply:
This is what the signature says on the click event

Reply:
The click event signature specifies it as InfernoMouseEvent, I think it should be fine, touch will probably also include the preventDefault method

Reply:
So there's no `InfernoClickEvent` that would cover both cases?

Reply:
Looks like there isn't. I'll test this on mobile anyway.